partials/pagination.hbs

```diff
@@ -0,0 +1,25 @@
+{{#match pages ">" 1}}
+    <nav class="pagination">
+
+        {{#if prev}}
+            <a href="{{page_url prev}}" class="button button--secondary button--small">
+                <span class="spacing-right-1" aria-hidden="true">&larr;</span>
```

Comment:
What about mobile screens? Will there be enough space to render both buttons including numbers on one line?

I would suggest quick solution and that is visually hide button labels for mobile...

Reply:
Good point! Fixed:

<img width="472" alt="obrazek" src="https://github.com/frontend-garden/frontend-garden-ghost-theme/assets/5614085/03bc1615-dca8-4c11-aa99-4516e42c568f">